docs/api/en/core/Geometry.html

```diff
@@ -266,7 +266,8 @@ <h3>[method:null dispose]()</h3>
 		</p>
 
 		<h3>[method:Geometry fromBufferGeometry]( [param:BufferGeometry geometry] )</h3>
-		<p>Convert a [page:BufferGeometry] to a Geometry.</p>
+		<p>Convert a [page:BufferGeometry] to a Geometry. <br />
+		The array used to store the vertices in the bufferGeometry is a non indexed array, so the resultant geometry may contain duplicated vertices. Use [page:mergeVertices] to remove them.</p>
```

Comment:
This belongs in a help site, or in the issues section -- not in the docs. The docs are not intended to be a tutorial.

Reply:
I think it's okay to add this text at this place. It's clearly a description of the internals of `.fromBufferGeometry()`

Reply:
giving the comment from @mrdoob  in #14857 I understood that it should be in the docs. Tell me where to put it and I will change it.
Although I think it can go in the docs, and I can see plenty of comments like this in the other functions, to help the users make good use of them, not only what they are, but how to use them.

Anyway, choose where to put it and I will move it there.

Reply:
I already told you where it belongs: in a help site or tutorial.

This addition will have no meaning to the average user. When, where, and why to use `megeVertices()` requires a more detailed explanation than is included here, and it does not belong in the description of `fromBufferGeometry()`.

Reply:
It seems like @mrdoob disagrees, though, so whatever he wants is OK. These are judgement calls, after all.